packages/legend-query-builder/src/graph-manager/QueryBuilderConfig.ts

```diff
@@ -33,6 +33,12 @@ export class QueryBuilderConfig {
    */
   TEMPORARY__enableGridEnterpriseMode = false;
 
+  /**
+   * This flag is to disable the "Show Virtual Assistant" button
+   * in Query Builder
+   */
+  TEMPORARY__disableVirtualAssistant = false;
```

Comment:
Based on your PR title, I think maybe we can introduce a boolean flag like TEMPORARY__disableInVSCodeExtension = true; then use this flag on buttons we want to hide

Reply:
Just spoke with Mauricio and instead of this approach, I will add a `disabled` flag to the `AssistantService` that controls the virtual assistant, and then add a `disable` function to the service so that we can simply disable the virtual assistant via that function when we create the `ApplicationStore` for the VS Code extension. This way we don't need to keep adding configuration specific to VS Code.